server.js

```diff
@@ -7,6 +8,14 @@ app.get('/', (req, res) => {
     res.send('Hello World!')
 })
 
+app.post('/transfer/:tokenId/:toAddress', async (req, res) => {
+    const { tokenId, toAddress } = req.params;
+    console.log("tokenId ", tokenId);
+    console.log("toAddress ", toAddress);
+    await nft.transferNFT(tokenId, toAddress);
+    return res.send("Transfered NFT");
```

Comment:
how will it handle failure scenario?
for e.g., the transfer failed to an address. what will this api return?